generators/server/templates/src/main/java/package/repository/UserRepository.java.ejs

```diff
@@ -121,7 +121,6 @@ import static <%= packageName %>.config.Constants.ID_DELIMITER;
     }
 _%>
 <%_ if ((databaseType === 'sql' && !reactive) || databaseType === 'mongodb' || databaseType === 'neo4j' || databaseType === 'couchbase') { _%>
-@Repository
```

Comment:
I don't think it should be removed ^^

Reply:
> 
> 
> I don't think it should be removed ^^

Excepted this last detail, LGTM!

Reply:
Maybe you are right. For sql we don't use a specific base repository. Will add it again.